superset-frontend/cypress-base/cypress/integration/dashboard/nativeFilters.test.ts

```diff
@@ -82,7 +82,7 @@ function prepareDashboardFilters(
   }).then(res => {
     const { body } = res;
     const dashboardId = body.result.id;
-    const allFilters: Record<string, any>[] = [];
+    const allFilters: Record<string, unknown>[] = [];
```

Comment:
I believe these are `Filters`:

```typescript
import { Filters } from '@superset-ui/core';

const allFilters: Record<string, Filters>[] = [];
```

Reply:
Hmm, I tried this but it gave me a "Module '"@superset-ui/core"' has no exported member 'Filter'." error.  I also didn't see anywhere else in `superset-frontend/cypress-base` that imports from `@superset-ui/core` – do you know if these modules are supposed to be separate for any reason?